examples/connext_dds/content_filtered_topic/c++11/cft_subscriber.cxx

```diff
@@ -9,46 +9,43 @@
  use the software.
  ******************************************************************************/
 
+#include <dds/sub/ddssub.hpp>
+#include <dds/core/ddscore.hpp>
+#include <rti/config/Logger.hpp>  // for logging
+
 #include "cft.hpp"
-#include <dds/dds.hpp>
-#include <rti/core/ListenerBinder.hpp>
-
-using namespace dds::core;
-using namespace dds::core::status;
-using namespace dds::core::policy;
-using namespace dds::domain;
-using namespace dds::topic;
-using namespace dds::sub;
-using namespace dds::sub::qos;
-
-class CftListener : public NoOpDataReaderListener<cft> {
-public:
-    void on_data_available(DataReader<cft> &reader)
-    {
-        // Take the available data
-        LoanedSamples<cft> samples = reader.take();
-
-        // Print samples by copying to std::cout
-        for (const auto &sample : samples) {
-            if (sample.info().valid()) {
-                std::cout << sample.data() << std::endl;
-            }
+#include "application.hpp"  // for command line parsing and ctrl-c
+
+int process_data(dds::sub::DataReader<cft> reader)
+{
+    int count = 0;
+    // Take the available data
+    dds::sub::LoanedSamples<cft> samples = reader.take();
+
+    // Print samples by copying to std::cout
```

Comment:
This comment is not correct. Maybe it was there before the code changed? You can delete it.